src/explorer.api/Models/NumericOverview.cs

```diff
@@ -0,0 +1,115 @@
+
+    using Aircloak.JsonApi;
+    using Aircloak.JsonApi.ResponseTypes;
+    using Explorer.Api.Models;
+
+    internal class NumericOverview : ColumnExplorer
+    {
+        internal NumericOverview(JsonApiSession apiSession, ExploreParams exploreParams)
+            : base(apiSession, exploreParams)
+        {
+        }
+
+        internal async IAsyncEnumerable<ExploreResult> Explore()
+        {
+            var queryParams = new QuerySpec(ExploreParams);
+
+            yield return new ExploreResult(ExplorationGuid, status: "waiting");
+
+            var queryResult = await ApiSession.Query<ResultRow>(
+                ExploreParams.DataSourceName,
+                queryParams.QueryStatement,
+                TimeSpan.FromMinutes(2));
+
+            var rows = queryResult.ResultRows;
+            Debug.Assert(
+                rows.Count() == 1,
+                $"Expected query {queryParams.QueryStatement} to return exactly one row.");
+
+            var stats = rows.First();
+
+            yield return new ExploreResult(ExplorationGuid, status: "complete", metrics: new List<ExploreResult.Metric>
+            {
+                new ExploreResult.Metric("Min")
+                {
+                    MetricType = AircloakType.Real,
+                    MetricValue = stats.Min,
+                },
+                new ExploreResult.Metric("Max")
+                {
+                    MetricType = AircloakType.Real,
+                    MetricValue = stats.Max,
+                },
+                new ExploreResult.Metric("Count")
+                {
+                    MetricType = AircloakType.Integer,
+                    MetricValue = stats.Count,
+                },
+                new ExploreResult.Metric("CountNoise")
+                {
+                    MetricType = AircloakType.Real,
+                    MetricValue = stats.CountNoise,
+                },
+            });
+        }
+
+        private class QuerySpec : IQuerySpec<ResultRow>
+        {
+            public QuerySpec(ExploreParams exploreParams)
+            {
+                TableName = exploreParams.TableName;
+                ColumnName = exploreParams.ColumnName;
+            }
+
+            public string QueryStatement => $@"
+                        select
+                            min({ColumnName}),
+                            max({ColumnName}),
+                            count(*),
+                            count_noise(*)
+                        from {TableName}";
+
+            public string TableName { get; }
+
+            public string ColumnName { get; }
+        }
+
+        private class ResultRow : IJsonArrayConvertible
+        {
+#pragma warning disable RCS1074 // Remove redundant constructor
+            // Need this for JsonArrayConverter constraint
+            public ResultRow()
+            {
+            }
+#pragma warning restore RCS1074 // Remove redundant constructor
+
+            internal double? Min { get; set; }
+
```

Comment:
We will eventually want to treat it as a number (to determine range bounds for example) so maybe decimal is better here. 

There are actually only two types of number that can be returned by Aircloak: `integer` or `real` so another possibility is to have two classes, one that uses int64 and another that uses doubles. <-- I think this might actually be a cleaner solution than the above.

What do you think?

Reply:
Yes, I believe that it would be better to have two different classes, one that uses int64 and the other using double.

Reply:
Ok i've done this in the latest push, broke it apart into two classes and extracted the query definition into its own namespace.